x/gamm/types/key.go

```diff
@@ -19,8 +19,14 @@ const (
 )
 
 var (
+	// TODO: deprecate
 	// KeyNextGlobalPoolId defines key to store the next Pool ID to be used.
 	KeyNextGlobalPoolId = []byte{0x01}
+	// KeyGammPoolCount defines key to store the count of gamm pools.
+	// Gamm pool count is equivalent to "next global pool id" that used to be
+	// in the gamm module. Since global pool id management has been moved
+	// to `swaprouter`, we convert this index to return the number of pools.
+	KeyGammPoolCount = KeyNextGlobalPoolId
```

Comment:
why not unify the usage when they're same values?

Reply:
I created a separate index for pool count, going to deprecate and remove pool id completely in a future release